### PR TITLE
Display joined missions on my profile

### LINKED ISSUE #2: [2pt]  Display joined missions in My profile - Filters
Render a list of all joined missions (use `filter()`) on the "My profile" page.

### PR DESCRIPTION
**Pull request by tobuya**
- Render all joined missions using `filter()` on the "My profile" page.

### CODE REVIEW

File: src/Redux/Missions/missions.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```
Comment (bernabasy) [inline src/Redux/Missions/missions.js:1]: please remove "eslint-disable" and follow camelcase. ( your first letter must be small and the first word of the second word must be small no space ) [look at here:](https://www.geeksforgeeks.org/how-to-convert-string-to-camel-case-in-javascript/)

File: src/Redux/Missions/missions.js [hunk 2]
```diff
@@ -11,7 +12,6 @@ const fetchMissions = createAsyncThunk(
   FETCHED_MISSIONS,
   async () => {
     const { data } = await axios.get(missionsAPI);
-    // eslint-disable-next-line camelcase
```
Comment (bernabasy) [inline src/Redux/Missions/missions.js:14]: You also repeat it here "eslint-disable-next-line" for camelcase, please take it into consideration!

**Review by bernabasy — APPROVED** [review]
@bernabasy  approved this pull request.
-----
### STATUS: APPROVED ✔️
Hi @tobuya ,

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 👍  Congratulations! 🎉💯🌟

wiserhood-cheers

---
 ### To highlight 🍾
- All liners are passing ✔️
- Good commit history ✔️
- Followed Github flow ✔️
- PR has a good title ✔️
## [Optional] suggestions:
- please look at the code below
-----
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏.

File: src/Redux/Missions/missions.js (hunk 2 above)
Comment (tobuya) [reply src/Redux/Missions/missions.js:14]: @bernabasy  thanks for the suggestions

**Pull request merged**